### DISCUSSION TITLE
does adding a thumbs up or other emoji to a comment on an issue count as "participating"?

### DISCUSSION THREAD

**Discussion by malcook** [community/community#21698] [New to GitHub]
<p>Does adding a thumbs up or other emoji to a comment on an issue count as “participating” for the purposes of notification?</p>
<p>If not, is the a reasonable change.  I hate to add a +1 comment just to stay notificed of progress on an issue I’m interested in.</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/malcook">@malcook</a>,</p>
<p>Creating emoji reactions does not subscribe you to an issue. If you want to subscribe without leaving a coment, there’s a subscribe button in the middle of the right column that you can use.</p>
